apps/vs-code-designer/src/app/commands/dataMapper/FxWorkflowRuntime.ts

```diff
@@ -89,60 +81,39 @@ async function createJsonFile(
 }
 
 async function waitForBackendRuntimeStartUp(url: string, initialTime: number): Promise<void> {
```

Comment:
This one now looks practically the same (except for a variable) to `waitForDesignTimeStartUp` in `startDesignTimeApi.ts`, might be worth coming back to it later to reuse the function!

Reply:
Yees! I will update the logic to reuse the `waitForDesignTimeStartUp` function. Thanks!